guide/getting-started/software.md

```diff
@@ -151,7 +155,7 @@ In between these first two categories fall decentralized exchanges. They may be
    layout = "float-right-desktop"
 %}
 
-A node connects to and participates in the bitcoin network to download and exchange transaction data. Some wallets have built-in nodes and some node software also supports wallet functionality. The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
+A network of computers called nodes maintains the blockchain. Individuals and groups running open-source Bitcoin software operate nodes. A node participates in the Bitcoin network by verifying activity on its blockchain. Some wallets have built-in nodes, and some node software supports wallet functionality. However, the minimum implementation of a node is syncing data and allowing other software a user has to access it.
```

Comment:
```suggestion
A node is simply a computer running a Bitcoin client. Nodes verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. Some wallets have built-in nodes, and some node software comes packaged with wallet functionality. However, the minimum implementation of a node is syncing data and allowing other software a user has to access it.
```

Could add some details about how different types of nodes exist (linking to nodes glossary page for more details).

Reply:
Can we just say `software`instead of `client? I am not sure if a newbie would understand the sentence you suggested.

Reply:
Agree with using the term software, I also find client confusing.

Reply:
Added both suggestions and a link https://github.com/BitcoinDesign/Guide/pull/437/commits/2a8065210dd4b367035f82cd0cbc35e444b39a95

Reply:
Yep software is good too